sdk/eventhub/event-hubs/test/receiver.spec.ts

```diff
@@ -210,16 +218,16 @@ describe("EventHub Receiver #RunnableInBrowser", function(): void {
       await client.createSender({ partitionId: partitionId }).send([ed]);
       debug("Sent the new message after creating the receiver. We should only receive this message.");
 
-      receiver = client.createReceiver(partitionId, {
-        beginReceivingAt: EventPosition.fromEnqueuedTime(pInfo.lastEnqueuedTimeUtc)
-      });
+      receiver = client.createReceiver(partitionId, EventPosition.fromEnqueuedTime(pInfo.lastEnqueuedTimeUtc));
       const data = await receiver.receiveBatch(10, 20);
       debug("received messages: ", data);
       data.length.should.equal(1, "Failed to received the expected single message");
       data[0].properties!.stamp.should.equal(uid);
     });
 
-    it("'after the particular sequence number' should receive messages correctly", async function(): Promise<void> {
+    it("'after the particular sequence number' should receive messages correctly #RunnableInBrowser", async function(): Promise<
```

Comment:
test name change due to merge conflicts?

Reply:
Yes, it seems to have gone back and forth a couple times, I'll fix this and the `bReceiver` in tests.